guides/source/association_basics.md

```diff
@@ -1506,7 +1506,7 @@ The `collection.where` method finds objects within the collection based on the c
 
 ##### `collection.exists?(...)`
 
-The `collection.exists?` method checks whether an object meeting the supplied conditions exists in the collection. It uses the same syntax and options as `ActiveRecord::Base.exists?`.
+The `collection.exists?` method checks whether an object meeting the supplied conditions exists in the collection. It uses the same syntax and options as [`ActiveRecord::Base.exists?`](active_record_querying.html#existence-of-objects).
```

Comment:
Can you hard wrap the modified line? Also I'd rather link to the API docs instead.